tests/unit/app_routes/utils/test_routes_utils_unit.py

```
@@ -11,11 +9,11 @@ def test_load_auth_payload_happy_path():
     class U:
         user_id = 9
         username = "user9"
-        access_token = "ak"  # noqa: S105
-        access_secret = "as"  # noqa: S105
+        access_token = "ak"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "access_token"

(S105)